showcase/covid19-galaxy.md

```diff
@@ -0,0 +1,149 @@
+  - the [COVID-19 Genomics UK Consortium](https://www.cogconsortium.uk/) (COG-UK; ENA Project accession: PRJEB37886)
+  - the [Portuguese network for SARS-CoV-2 genomics](https://insaflu.insa.pt/covid19/) (ENA Project accession: PRJEB47340)
+  - the Estonian national sequencing initiatives (KoroGeno-EST-3 and KoroGeno-EST-2022; see [dedicated showcase](https://www.infectious-diseases-toolkit.org/showcase/korogenoest.html))
+
+In addition, we provide the possibility to request analysis of samples of particular interest via pull requests against a dedicated [GitHub repository](https://github.com/usegalaxy-eu/sars-cov-2-processing-requests).
+
+### WorkflowHub
+
+The [WorkflowHub registry](https://workflowhub.eu/) for scientific computational workflows provides access to hundreds of workflows with defined releases, among them several [Galaxy workflows for SARS-CoV-2 genome analysis](https://workflowhub.eu/workflows?filter%5Bproject%5D=33&filter%5Bquery%5D=covid).
+
+These are provided and/or reviewed by the [IWC](https://github.com/galaxyproject/iwc), a subgroup within the Galaxy Community concerned with maintaining high-quality Galaxy Workflows and who collaborates closely with WorkflowHub on FAIR workflow issues.
+
+Most IWC SARS-CoV-2 analysis workflows follow a modular design decision, i.e. there are separate workflows for discovering viral mutations from raw sequencing data obtained from different sequencing platforms and protocols, for reporting and visualising these mutations within and across samples, and for generating viral consensus genomes.
+
+This has the advantage that users can combine analysis modules according to their specific needs.
+
+### Galaxy Europe
+
+The [European Galaxy server](https://usegalaxy.eu) provides free access to powerful publicly-funded compute infrastructure and thousands of bioinformatics tools.
+
+In the showcase we download ENA-hosted or user-requested sequencing data to the Galaxy server and process it with the workflows from WorkflowHub.
+
+### Archive
+
+The archive is provided by the [Centre for Genomic Regulation](https://www.crg.eu/) (CRG).
+
+For every sample processed through Galaxy workflow runs, we are exporting key results files
+(reads mapped to the viral reference sequence, variant calls, consensus sequences and tabular reports)
+to a public archive so that consumers can discover, access and reuse it.
+
+We are also maintaining a [provenance JSON file](https://galaxyproject.org/projects/covid19/data/#information-about-every-analysis-run) in this archive
+that makes it possible to reconstruct the full data processing history of each sample.
+
+You can access all the data by logging in to: `ftp://xfer13.crg.eu` as User: `FTPuser` Password: `FTPusersPassword`
+
+### Viral Beacon project
+
+In the showcase the [Viral Beacon project](https://covid19beacon.crg.eu/) acts as a consumer of the data and provides visualisations.
```

Comment:
```suggestion
In the showcase the [ CRG Viral Beacon](https://covid19beacon.crg.eu/) acts as a consumer of the data and provides visualisations.
```
Added specific name of the tool to avoid confusion with the Beacon project in general

Reply:
Also adding CRG Viral Beacon to the tools and resources table